src/tests/flyer.test.ts

```diff
@@ -294,4 +293,32 @@ describe('deleteFlyer tests', () => {
     const deleteFlyerResponse = await FlyerRepo.deleteFlyer('64811792f910705ca1a981f8');
     expect(deleteFlyerResponse).toBeNull();
   });
-});
\ No newline at end of file
+});
+
+describe('getAllFlyerCategories tests', () => {
+  test('Ensure only unique CategorySlugs across flyers in collection are returned', async () => {
+    const flyerSlugs = ['Academic', 'Dance', 'Dance'];
+    const generatedFlyers: Flyer[] = [];
+    for (const slug of flyerSlugs) {
+      generatedFlyers.push(...(await FlyerFactory.createSpecific(1, { categorySlug: slug })));
```

Comment:
Using concat is cleaner than trying to unpack the array:

```suggestion
      generatedFlyers.concat(await FlyerFactory.createSpecific(1, { categorySlug: slug }));
```